Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -457,11 +483,22 @@ extension PlaceCarouselViewController: LocationMonitorDelegate {
     func locationMonitor(_ locationMonitor: LocationMonitor, userDidExitCurrentLocation location: CLLocation) {
         locationMonitor.refreshLocation()
     }
+
+    func locationMonitor(_ locationMonitor: LocationMonitor, didFailInitialUpdateWithError error: Error) {
+        // Note: actually, at this point, if the location comes, `didUpdateLocation` will be called.
+        presentNoLocationAlert()
+    }
 }
 
 extension PlaceCarouselViewController: PlacesProviderDelegate {
     fileprivate func showErrorMessageIfNoPlaces() {
         if self.places.isEmpty {
+            let hasLocation = locationMonitor.getCurrentLocation() != nil
+            guard hasLocation else {
+                presentNoLocationAlert()
```

Comment:
afaik, the only error is shown below this but it's possible we'll add one.

Is it simple to check for another alert?